winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/CliClient.java

```diff
@@ -23,18 +20,18 @@ public static void main(String[] args) throws Exception{
         String password = args[2];
         String cmd = args[3];
 
-        WinRmClient client = WinRmClient.builder(endpoint)
+        WinRmTool client = WinRmTool.Builder.builder(endpoint, username, password)
                 .disableCertificateChecks(true)
-                .credentials(username, password)
                 .workingDirectory("C:\\")
 //                .environment(env)
                 .build();
         int exitCode = 999;
         try {
-            exitCode = client.command(cmd, new PrintWriter(new OutputStreamWriter(System.out)), new PrintWriter(new OutputStreamWriter(System.err)));
```

Comment:
This, at least in theory, allows for interleaving the stdout & stderr output, similar to how you'd see it in the console. Also the output will be visible as soon as the poll call completes, not at the end of the command. So not convinced it's worth using `WinRmTool` directly here.